wallet-api/src/Wallet/UTXO.hs

```diff
@@ -144,18 +142,35 @@ especially because we only need one direction (to binary).
 
 -}
 
+-- | We use structured JSON where its easy but use Encoding where its not
```

Comment:
We can be more specific: I think we are only using it for the serialized PLC code.